commands/create_command.py

```diff
@@ -0,0 +1,29 @@
+import json
+import os
+from keys import keys
+from commands.abstract_command import ICommand
+
+class CreateCommand(ICommand):
+
+    def __init__(self):
+        with open('schema.json','r') as schema:
+            data = json.load(schema)
+        self.excute(data)
+        
+    def excute(schema_path,data):
```

Comment:
I see you don't make use of schema_path, so remove it and work with the loaded data directly.

Reply:
+1